picard/mbxml.py

```diff
@@ -195,7 +195,7 @@ def artist_credit_to_metadata(node, m, release=False):
         m["musicbrainz_albumartistid"] = ids
         m["albumartist"] = artist
         m["albumartistsort"] = artistsort
-        m["~albumartists"] = artists
```

Comment:
Maybe we'll want to keep `~albumartists` (alongside `albumartists`) for the 1.4.x releases, marking it as deprecated and remove for 2.0?